src/lib.rs

```diff
@@ -24,7 +24,7 @@ extern crate quickcheck;
 
 use std::io::Write;
 
-pub use color::ColorType::{self, Gray, GrayA, Palette, RGB, RGBA, BGR, BGRA};
```

Comment:
Not sure if this is the right move. It isn't really idiomatic to `pub use` a bunch of enum variants, but removing them could unnecessarily break people's code.

Reply:
`libstd` does the same by using `Option::*` so that `Some()` and `None` works. I'd say it's idiomatic enough where useful and unambiguous.

Reply:
The prelude is a bit different though isn't it? The Some and None variants are automatically in scope whereas bringing bring in these variants would typically involve either `use image::{RGB, RGBA, ...}` or `use image::ColorType::*` which are basically the same (people generally don't pull in entire crates with a wildcard, with the exception of special "prelude" submodules).

Reply:
Fair points, and it does clobber the documentation a little.